velox/functions/lib/aggregates/noisy_aggregation/sketch/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+velox_add_library(velox_sketch SfmSketch.cpp)
```

Comment:
> velox_sketch

CMake target names must be unique. Hence, they need to include the full directory path: velox_functions_lib_aggregates/noisy_aggregatio_sketch.

velox/functions/lib contains utilities that are shared between different dialects (Presto and Spark). SfmSketch appears to be Presto-specific, hence, doesn't belong here.